cmd/configschema/go.mod

```
@@ -330,6 +330,7 @@ require (
 	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/googlecloudspannerreceiver v0.47.0 // indirect
 	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/hostmetricsreceiver v0.47.0 // indirect
 	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/influxdbreceiver v0.47.0 // indirect
+		github.com/open-telemetry/opentelemetry-collector-contrib/receiver/iisreceiver v0.47.0 // indirect
```

```suggestion
	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/iisreceiver v0.47.0 // indirect
```
Not sure why but this appears to be over-intended.

---

Updated